### ISSUE TITLE
Select popover receives listbox attributes

### ISSUE THREAD

**Issue by marcospassos** [ariakit/ariakit#1549]
### Current behavior

As initially discussed [here](https://github.com/ariakit/ariakit/discussions/1547), the `<SelectPopover/>` renders attributes that should be handled by the `<SelectList/>`, which causes issues for screenreaders.

### Steps to reproduce the bug

1. Open sandbox: https://codesandbox.io/s/funny-brook-6q8z9c
2. Inspect the popover
3. See `aria-multiselectable="true"` even though `composite={false}`

### Expected behavior

The popover should only render popover-specific attributes.

### Workaround

```jsx
<SelectPopover aria-multiselectable={undefined} />
```
